java/bulk-export/bulk-export-core/pom.xml

```diff
@@ -23,4 +23,48 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>bulk-export-core</artifactId>
+
+    <dependencies>
+        <!-- Sleeper dependencies -->
+        <dependency>
+            <groupId>sleeper</groupId>  
+            <artifactId>core</artifactId>
+            <version>${project.parent.version}</version>   
+        </dependency>
```

Comment:
You could make the dependency check happy for now by setting this to runtime scope.

Reply:
I had to set this to compile for it to work.